Source/Engine/Engine.hpp

```diff
@@ -76,7 +76,5 @@ void Engine::AddEventHandler(EventType type, std::function<void(const T&)> handl
 {
     std::vector<EventHandler>& eventHandlers = eventMap[type];
     eventHandlers.emplace_back([handler](std::any argument)
-        {
-            handler(std::any_cast<T>(argument));
-        });
+            { handler(std::any_cast<T>(argument)); });
```

Comment:
Lets try to fix Lambdas formatting: avoid such shrinking and keep offset of code block